extensions/wikia/AdEngine/js/wad/wadRecRunner.js

```diff
@@ -0,0 +1,22 @@
+/*global define*/
+define('ext.wikia.adEngine.wad.wadRecRunner', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.wad.btRecLoader',
+	'wikia.log'
+], function (adContext, btRecLoader, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.wad.wadRecRunner';
+
+	function init() {
+		log('WAD recovery module initialized', 'debug', logGroup);
+
+		if (adContext.get('opts.wadBT')) {
```

Comment:
Why do we have two different flags/instant globals for it?
opts.babRecovery and opts.wadBT

Reply:
Go it. babRecovery wraps wad with babDetection, right?

Reply:
That's right.